frontend/apps/web/app/(mgmt)/[account]/runs/[id]/page.tsx

```diff
@@ -108,6 +109,39 @@ export default function Page({ params }: PageProps): ReactElement {
   const [isRetrievingRunContext, setIsRetrievingRunContext] =
     useState<boolean>(false);
 
+  const [duration, setDuration] = useState<string>('');
+
+  useEffect(() => {
+    let timer: NodeJS.Timeout;
+    if (jobRun?.startedAt && jobRun?.status === JobRunStatusEnum.RUNNING) {
+      const updateDuration = () => {
+        const now = new Date();
+        const startTime = jobRun?.startedAt?.toDate();
+        if (startTime) {
+          const diffInSeconds = Math.floor(
+            (now.getTime() - startTime.getTime()) / 1000
+          );
+          const minutes = Math.floor(diffInSeconds / 60);
+          const seconds = diffInSeconds % 60;
+          // handle plural minutes and seconds and set the duration
+          setDuration(formatDuration({ minutes: minutes, seconds: seconds }));
```

Comment:
Why can't this just be:
```suggestion
          setDuration(getDuration(new Date(), startTime);
```